### ISSUE TITLE
Ethiopian Kale Leaf Image Analysis

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#471]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Ethiopian Kale Leaf Image Analysis
:red_circle: **Aim** : Create a data analysis project to analyze the dataset provided to you.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/damtewmekash/ethiopiankale
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by CoderOMaster** [comment]
Full name : Keshav Arora 
GitHub Profile Link : https://github.com/CoderOMaster 
Participant ID (If not, then put NA) :
Role: JWOC 2024 Mentee
Approach for this Project:
Various Deep learning techniques can be implemented using CNN,RNN and architectures like VGG,Resnet, inception for transfer learning.
Data augmentation will be implemented if needed.
Regards

**Comment by akanksha-2002** [comment]
Hi, @abhisheks008  ! I would like to take up this issue. 
Full name : Akanksha Bhimte
GitHub Profile Link: https://github.com/akanksha-2002
Participant ID (If not, then put NA) :
Approach for this Project: 
1. EDA
2. Data preprocessing for analysis
3. Employing models such as CNN, GoogLeNet, Transfer Learning Models and Ensemble models to compare their performance and find the best fit. 
4. Evaluation Metrics
5. Fine-tuning and Optimization with different architectures, hyperparameters, and regularization techniques.


What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)
JWOC

**Comment by abhisheks008** [comment]
@akanksha-2002 and @CoderOMaster wait for the program to start officially. Once it is done, issues will be assigned to you accordingly.

**Comment by CoderOMaster** [comment]
Hi @abhisheks008
Full name : Keshav Arora
GitHub Profile Link : https://github.com/CoderOMaster
Participant ID (If not, then put NA) :
Role: JWOC 2024 Mentee
Approach for this Project:
Various Deep learning techniques can be implemented using CNN,RNN and architectures like VGG,Resnet, inception for transfer learning.
Data augmentation will be implemented if needed.
Regards

**Comment by abhisheks008** [comment]
Issue assigned to  you @akanksha-2002

**Comment by Tanishka023** [comment]
Hey @abhisheks008 Sir, please assign this issue to me under SSOC'24, I really want to work on this.

**Comment by abhisheks008** [comment]
> Hey @abhisheks008 Sir, please assign this issue to me under SSOC'24, I really want to work on this.

One issue at a time. Please do not spam.

**Comment by Tanishka023** [comment]
Heyy @abhisheks008
Full name : Tanishka Bhalla 
GitHub Profile Link : https://github.com/Tanishka023
Participant ID (If not, then put NA) : NA
Role: SSoC Season 3
Approach for this Project:
After processing the images, various ML models can be used such as SVM, KNN, LR, Random forests

**Comment by abhisheks008** [comment]
> Heyy @abhisheks008 Full name : Tanishka Bhalla GitHub Profile Link : https://github.com/Tanishka023 Participant ID (If not, then put NA) : NA Role: SSoC Season 3 Approach for this Project: After processing the images, various ML models can be used such as SVM, KNN, LR, Random forests

One issue at a time.

**Comment by tanuj437** [comment]
@abhisheks008 The dataset link provided in this issue is not available as page can't be found showing,
If there is any other link for it do provide i am up for working over it

Full name : Tanuj Saxena
GitHub Profile Link : https://github.com/tanuj437
Participant ID (If not, then put NA) : NA
Approach for this Project : EDA firstly then starting with Basic Algorithm like Logistic regression, SVM, K-NN ,Random Forest, Gradient boosting, then can go with Neural Networks like CNN
What is your participant role? SSOC'24 season 3

**Comment by abhisheks008** [comment]
Implement 6-7 models for this project.

Assigned @tanuj437
